.github/pull_request_template.md

```diff
@@ -10,8 +10,7 @@ Include the following checklist in your PR:
 * [ ] Any changes that could be relevant to users [have been recorded in the changelog](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#changelog).
 * [ ] The documentation has been updated, if necessary.
 * [ ] [Manual QA notes](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#qa-notes) have been included.
-
-Bonus points for added automated tests!
+* [ ] Tests have been added. (Ask for help if you don’t know how to write them! Ask for an exemption if tests are too complex for too little coverage!)
```

Comment:
Maybe italicize the stuff in parens so that it reads as mate right away?

Reply:
Sure!